components/suggestion/suggestion_box.jsx

```diff
@@ -202,18 +201,14 @@ export default class SuggestionBox extends React.Component {
             return;
         }
 
-        this.setState({focused: false});
-
         if (UserAgent.isIos() && !e.relatedTarget) {
-            // iOS doesn't support e.relatedTarget, so we need to use the old method of just delaying the
-            // blur so that click handlers on the list items still register
-            if (this.presentationType !== 'date' || this.props.value.length === 0) {
-                this.handleEmitClearSuggestions(MIN_TIME_BEFORE_CLICK);
-            }
-        } else {
-            this.handleEmitClearSuggestions();
+            return;
```

Comment:
If that's okay, can you add a comment here explaining the different behaviour on iOS, Saturn?

Reply:
Thanks @hmhealey, PR updated with comment.